cmd/kube-proxy/app/server_linux_test.go

```diff
@@ -114,122 +114,108 @@ func Test_getLocalDetector(t *testing.T) {
 		mode         proxyconfigapi.LocalMode
 		config       *proxyconfigapi.KubeProxyConfiguration
 		family       v1.IPFamily
-		expected     proxyutiliptables.LocalTrafficDetector
+		expected     proxyutil.LocalTrafficDetector
 		nodePodCIDRs []string
-		errExpected  bool
 	}{
 		// LocalModeClusterCIDR
 		{
-			name:        "LocalModeClusterCIDR, IPv4 cluster",
-			mode:        proxyconfigapi.LocalModeClusterCIDR,
-			config:      &proxyconfigapi.KubeProxyConfiguration{ClusterCIDR: "10.0.0.0/14"},
-			family:      v1.IPv4Protocol,
-			expected:    resolveLocalDetector(t)(proxyutiliptables.NewDetectLocalByCIDR("10.0.0.0/14")),
-			errExpected: false,
+			name:     "LocalModeClusterCIDR, IPv4 cluster",
+			mode:     proxyconfigapi.LocalModeClusterCIDR,
+			config:   &proxyconfigapi.KubeProxyConfiguration{ClusterCIDR: "10.0.0.0/14"},
+			family:   v1.IPv4Protocol,
+			expected: proxyutil.NewDetectLocalByCIDR("10.0.0.0/14"),
 		},
 		{
-			name:        "LocalModeClusterCIDR, IPv6 cluster",
-			mode:        proxyconfigapi.LocalModeClusterCIDR,
-			config:      &proxyconfigapi.KubeProxyConfiguration{ClusterCIDR: "2002:0:0:1234::/64"},
-			family:      v1.IPv6Protocol,
-			expected:    resolveLocalDetector(t)(proxyutiliptables.NewDetectLocalByCIDR("2002:0:0:1234::/64")),
-			errExpected: false,
+			name:     "LocalModeClusterCIDR, IPv6 cluster",
+			mode:     proxyconfigapi.LocalModeClusterCIDR,
+			config:   &proxyconfigapi.KubeProxyConfiguration{ClusterCIDR: "2002:0:0:1234::/64"},
+			family:   v1.IPv6Protocol,
+			expected: proxyutil.NewDetectLocalByCIDR("2002:0:0:1234::/64"),
 		},
 		{
-			name:        "LocalModeClusterCIDR, IPv6 cluster with IPv6 config",
-			mode:        proxyconfigapi.LocalModeClusterCIDR,
-			config:      &proxyconfigapi.KubeProxyConfiguration{ClusterCIDR: "10.0.0.0/14"},
-			family:      v1.IPv6Protocol,
-			expected:    proxyutiliptables.NewNoOpLocalDetector(),
-			errExpected: false,
+			name:     "LocalModeClusterCIDR, IPv6 cluster with IPv6 config",
```

Comment:
Not introduced by the PR, but should it be "IPv6 cluster with IPv4 config"?

Reply:
heh. yep.